### PR TITLE
Fix jar conflics

### PR DESCRIPTION
**Pull request by drEfrati**
- Remove the 2 jars in the stellar part, for the crypto jar just add a new . dependency in gradle instead of the jar and for the commons just copy the necessary classes to our own package
- Change some dependencies in tests because of the previous  jars removal- 
- Also rename the ed25519-sha-512 string in KeyPair class because of the crypto jar dependency change.

* invoked all the tests and they passed

### CODE REVIEW

File: kin-base/src/main/java/kin/base/KeyPair.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
  */
 public class KeyPair {
 
-  private static final EdDSANamedCurveSpec ed25519 = EdDSANamedCurveTable.getByName("ed25519-sha-512");
+  private static final EdDSANamedCurveSpec ed25519 = EdDSANamedCurveTable.getByName("ed25519");
```
Comment (yosriz) [inline kin-base/src/main/java/kin/base/KeyPair.java]: use the library const `EdDSANamedCurveTable.ED_25519_CURVE_SPEC` see https://github.com/stellar/java-stellar-sdk/commit/3826aa224bad802c4524858c75f4c2e65ba5ee9f

**Comment by drEfrati** [comment]
@yosriz Also changed the travis to what i think you meant

**Comment by yosriz** [comment]
@drEfrati there are no changes since my review

**Comment by drEfrati** [comment]
@yosriz because of some problems i made a new pr so you will see the new commits

**Pull request closed without merging**